examples/nas/enas/macro.py

```diff
@@ -0,0 +1,83 @@
+import torch.nn as nn
+
+from nni.nas.pytorch import mutables
+from ops import FactorizedReduce, ConvBranch, PoolBranch
+
+
+class ENASLayer(mutables.MutableScope):
```

Comment:
when we need to implement from mutables.MutableScope?

Reply:
For now, you can read the docstring of MutableScope. We might write a tutorial later.

Reply:
@xuehui1991 MutableScope can be viewed as namespace, in order for trainer to get the hierarchical structure.